subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/internal/BuildFailedListener.kt

```diff
@@ -0,0 +1,13 @@
+package com.avito.android.build_verdict.internal
+
+import com.avito.android.build_verdict.internal.writer.BuildVerdictWriter
+
+internal class BuildFailedListener(
```

Comment:
why do you think this abstraction better than just provide list of writers to every failure listener?

Reply:
Right now we just dump the verdict to a file, but I guess in the future we might decide to write it to the logs, etc. So I want a verdict callback and decide what I will do there.